test/cookie_coder_test.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "test_helper"
+
+describe Identity::CookieCoder do
+  secret = "Pw/Xv58CsV8FVUQldtbHRZBoGqv8pJ1u55JdlhsRp9A="
+  another_secret = "r1auRylYrR9WM3dhRkfXRwQ5nxcNGR2lBkCLgXl94gU="
+
+  it "encryptes a cookie, then descrypts a cookie" do
+    data = {
+      user: {
+        id: "1234",
```

Comment:
Is there a reason why the user ID is being placed in the cookie?  I wonder about security concerns.  Do we surface it anywhere else ? (I realize we're applying some encryption here, but still).